lib/vagrant-service-manager/services/openshift.rb

```diff
@@ -0,0 +1,24 @@
+module Vagrant
+  module ServiceManager
```

Comment:
I even wonder why we talk about Docker as a service. OpenShift, Kubernetes, Atomic etc are for me the services. Docker is needed for all of them afaik. A ADB/CDK w/o Docker does not make any sense whatsoever. My standpoint is that Docker configuration is always implicit and just happens and depending whether additional services are enabled there might be additional  Docker options set, but overall Docker is just a process running on the VM. I think taking this POV makes things less ambiguous.